src/pages/resultsView/SurvivalStoreHelper.ts

```diff
@@ -36,14 +39,16 @@ export function getPatientSurvivals(
                             monthsClinicalData.value
                         ) &&
                         !Number.isNaN(Number(monthsClinicalData.value)) &&
-                        Number(monthsClinicalData.value) >= 0
+                        Number(monthsClinicalData.value) >= 0 &&
```

Comment:
lets briefly discuss this condition.  at very least it needs a comment to explain that if we have no entryMonth data then we assume entryMonth = 0

Reply:
Yes, I agree. I will add a comment here